### PR TITLE
[Feat]:  auth session implementation for achievements

### PR DESCRIPTION
**Pull request by eliobricenov**
With this PR we introduce a new hook that uses the cookies from the `persist:auth` partition and gets the session object from `DEV_PORTAL/api/auth/session`.

Thanks to react-query, and a new event that's emitted from the devportal to the webview host when the users signs in, we can invalidate the query and re-fetch the session object.

How to test: https://app.qase.io/case/HC-323

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by eliobricenov** [comment]
How to clear your webiew cookies: https://www.loom.com/share/5fac7b9246a44552a7cf09a540e6a75f?sid=12c7b120-ae0d-45f5-adc7-37904273b5b8

File: src/common/constants.ts [hunk 1]
```diff
@@ -8,4 +8,6 @@ export const METAMASK_SNAPS_URL = 'https://snaps.metamask.io/'
 export const METAMASK_SNAPS_EXTERNAL_URL = 'https://metamask.io/snaps/'
 export const METAMASK_PORTFOLIO_URL = 'https://portfolio.metamask.io/'
 
-export const DEV_PORTAL_URL = 'https://developers.hyperplay.xyz/'
+// will be updated to prod when the dev portal PR is merged
+export const DEV_PORTAL_URL =
```
Comment (BrettCleary) [inline src/common/constants.ts:12]: making a comment as a reminder here

**Review by BrettCleary — APPROVED** [review]
:fire: :fire: :fire:

**Review by nyghtstalker — APPROVED** [review]
I was able to get it all connected and see my Steam achievements being displayed on the Achievement tab. Looks and works great! Awesome job!!! 

image

**Review by red-game-dev — APPROVED** [review]

File: src/common/constants.ts (hunk 1 above)
Comment (eliobricenov) [reply src/common/constants.ts]: update to prod, thank!

**Pull request merged**